### PR TITLE
396 radspec message for new Allocation

### PR DESCRIPTION
**Pull request by topocount**
It appears that forwarded notice messages only appear in the Panel UI when they are immediately followed by Parameter Values. This PR conforms to that constraint and the message is appearing in the UI after I :

1. Create a new Allocations account
2. Create a new Allocation payout

and the forwarding notification appears in the side panel.

@stellarmagnet Please let me know if you'd like me to tweak the language on any of these

### CODE REVIEW

**Comment by Quazia** [comment]
Does adding params not work in the notice? If @stellarmagnet is okay with this amount of information it's fine with me, there's nothing technically wrong just not sure it's informative enough.

File: apps/allocations/contracts/Allocations.sol [hunk 1]
```diff
@@ -162,16 +162,21 @@ contract Allocations is AragonApp, Fundable {
...
-    function fund(uint256 id) external payable { 
+    function fund(uint256 id) external payable {
         Payout storage payout = payouts[id];
         require(!payout.informational);
         payout.balance = payout.balance.add(msg.value);
         emit FundAccount(id);
     }
 
+    /**
+    * @dev This is the function distributes the payouts to the candidates in accordance with the distribution values
```
Comment (Quazia) [inline apps/allocations/contracts/Allocations.sol]: Wording here is wrong, maybe "function that" or "This function".

File: apps/allocations/contracts/Allocations.sol [hunk 2]
```diff
@@ -207,10 +212,14 @@ contract Allocations is AragonApp, Fundable {
     *      to be called by a RangeVote (options get weird if it's not)
     *      but for our use case the “SET_DISTRIBUTION_ROLE” will be given to
     *      the RangeVote.
-    * @notice Sets the distribution for the given `payoutId` using an the
```
Comment (Quazia) [inline apps/allocations/contracts/Allocations.sol]: Are the params not working?

File: apps/allocations/contracts/Allocations.sol [hunk 3]
```diff
@@ -144,11 +144,11 @@ contract Allocations is AragonApp, Fundable {
     * @dev This is the function that sets up who the candidates will be, and
     *      where the funds will go for the payout. This is where the payout
     *      object needs to be created in the payouts array.
-    * @notice Create a new account so you can begin creating allocations.
+    * @notice Create a new account used for creating allocations
```
Comment (stellarmagnet) [inline apps/allocations/contracts/Allocations.sol]: LGTM

File: apps/allocations/contracts/Allocations.sol [hunk 4]
```diff
@@ -162,16 +162,21 @@ contract Allocations is AragonApp, Fundable {
...
         payout.balance = payout.balance.add(msg.value);
         emit FundAccount(id);
     }
 
+    /**
+    * @dev This function distributes the payouts to the candidates in accordance with the distribution values
+    * @notice Send payout amounts to the candidates in accordance with the distribution proportions
+    * @param _payoutId Any relevent label for the payout
+    */
     function runPayout(uint256 _payoutId) external payable isInitialized auth(EXECUTE_PAYOUT_ROLE) returns(bool success) {
```
Comment (stellarmagnet) [inline apps/allocations/contracts/Allocations.sol]: Where in the UI is this?

File: apps/allocations/contracts/Allocations.sol [hunk 5]
```diff
@@ -207,10 +212,14 @@ contract Allocations is AragonApp, Fundable {
...
-    * param _supports The Array of all support values for the various candidates
+    * @notice Sets the distribution proportion for this allocation
+    * @param _candidateAddresses Array of candidates to be allocated a portion of the payouut
+    * @param _supports The Array of all support values for the various candidates. These values are set in range voting
+    * @param _payoutId The Account used for the payout
+    * @param _informational boolean used to indicate whether and funds will be trnsacted onchain
+    * @param _recurring boolean used to indicate whether this is a recurring or one-time payout
+    * @param _period time interval between each recurring payout
+    * @param _amount The quantity of funds to be allocated
     */
```
Comment (stellarmagnet) [inline apps/allocations/contracts/Allocations.sol]: Where in the UI is this?

File: apps/allocations/contracts/Allocations.sol (hunk 4 above)
Comment (topocount) [reply apps/allocations/contracts/Allocations.sol]: This shows up when you begin the distribute payout/runpayout transaction

File: apps/allocations/contracts/Allocations.sol (hunk 5 above)
Comment (topocount) [reply apps/allocations/contracts/Allocations.sol]: This notic appears when you click "New Allocation" on the Allocation account card

Comment (topocount) [reply apps/allocations/contracts/Allocations.sol]: Line 215 in the new changes is what will be displayed: https://github.com/AutarkLabs/planning-suite/pull/470/files/773673273cf237aed9ea8387398b1a1e877c6e00#diff-d4d9bf65696fe98cabb21de714647055R215

**Pull request merged**